City-of-Pori/pori-kada
# PORI-X Fixed weights on the link to content items in carousel

To test:

1. drush fra -y && drush cc all
2. Go to front page and see if the slideshow works correctly.
3. Go to theme main page and see if the slideshow works correctly.
